scripts/cc-cli/src/commands/chill.ts

```diff
@@ -24,6 +25,13 @@ async function chillAction(options: OptionValues) {
   const { api } = await newApi(options.url);
 
   const controllerSeed = getSeedFromOptions(options);
+  const controllerKeyring = initKeyringPair(controllerSeed);
+  const controllerAddress = controllerKeyring.address;
+
+  const controllerStatus = await getStatus(controllerAddress, api);
+  const stashStatus = await getStatus(controllerStatus.stash, api);
+
+  requireStatus(stashStatus, "validating");
 
   console.log("Creating chill transaction...");
```

Comment:
I got a traceback which is related to the generated data types, but not to checking the status.

FTR this account isn't validating, nor bonded.

```
$ node dist/index.js chill -u ws://127.0.0.1:9946 -s "twice wash veteran endorse water orbit fit casino flame provide cancel peace"
2023-06-21 20:28:34        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-21 20:28:34        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/@polkadot/types-create/cjs/create/type.js:71
    firstError = new Error(`createType(${type}):: ${error.message}`);
                 ^

Error: createType(AccountId):: Decoding None: Invalid decoded address checksum
    at createTypeUnsafe (/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/@polkadot/types-create/cjs/create/type.js:71:18)
    at TypeRegistry.createType (/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/@polkadot/types/cjs/create/registry.js:288:46)
    at /home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/@polkadot/api-derive/cjs/staking/query.js:69:55
    at Array.map (<anonymous>)
    at /home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/@polkadot/api-derive/cjs/staking/query.js:69:33
    at /home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/rxjs/dist/cjs/internal/operators/switchMap.js:17:35
    at OperatorSubscriber._this._next (/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:33:21)
    at Subscriber.next (/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
    at ReplaySubject._subscribe (/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/rxjs/dist/cjs/internal/ReplaySubject.js:54:24)
    at Observable._trySubscribe (/home/gluwa/creditcoin-2.0/creditcoin-substrate/scripts/cc-cli/node_modules/rxjs/dist/cjs/internal/Observable.js:41:25)

Node.js v18.1.0
```

Reply:
Ohh, got it. Seems like when trying to get that controller's stash status (the actual validator status) its throwing an unhanlded error because its trying to use `None` as an `AddressId`. Should check if the account is a controller (has a Stash?) before attempting to get the Stash status and print a proper error.